Src/Common/RequestResponseHeaders.cs

```diff
@@ -8,12 +8,12 @@ internal static class RequestResponseHeaders
         /// <summary>
         /// Source instrumentation header that is added by an application while making http requests and retrieved by the other application when processing incoming requests.
         /// </summary>
-        public const string SourceInstrumentationKeyHeader = "x-ms-request-source-ikey";
```

Comment:
I have a meeting with @lmolkova tomorrow around switching to `Request-ID` instead of `x-ms-request-id`. So maybe for consistency we will need to drop `x-ms-` here as well. Also I like `request-source`, `response-target` doesn't sound good. `request-target` makes more sense when pronouncing it in a context. Saying response target it's unclear whether it is where response originated from or to...

Alternatively we can do `request-context` and `response-context` headers similar to `correlation-context`. We can follow the format of `appId=XXX, roleInstance = YYY`.

When do we need to finalize on this schedule wise?